### PR TITLE
make_2d_plots.py now handles relative proportions explained correctly

### LINKED ISSUE #1884: a bug of fraction variance in make_2d_plots.py
It mistakenly shows fraction as percentage. see the post https://groups.google.com/d/msg/qiime-forum/30-EvCrQotA/0Mbqc0qJ8-UJ
It occurs in 1.9 rc2 and rc1

### PR DESCRIPTION
**Pull request by ebolyen**
fixes #1884 

Uses the same interface and behaviour as emperor.

@ElDeveloper, would you be able to review this one?

### CODE REVIEW

**Comment by qiime-r2d2** [comment]
Build results will soon be (or already are) available at: http://ci.qiime.org/job/qiime-github-pr/1568/

File: scripts/make_2d_plots.py [hunk 1]
```diff
@@ -143,6 +143,11 @@
         action='store_true',
         help='Generate the scree plot [default: %default]',
         default=False),
+    make_option('--pct_variation_below_one',
+                action="store_true",
+                help='Allow the percent variation explained by the axis to be below one. '
```
Comment (ElDeveloper) [inline scripts/make_2d_plots.py]: axis->axes?

**Comment by ElDeveloper** [comment]
This looks good to me, but do you think we could add another test case to check for this new option?

File: scripts/make_2d_plots.py (hunk 1 above)
Comment (ebolyen) [inline scripts/make_2d_plots.py]: This was copied from emperor, a similar misspelling may exist there.

**Comment by ebolyen** [comment]
@ElDeveloper Unfortunately, this code is not maintainable enough to robustly test... The only testing that happens on the function that was modified checks to see if a file was created and nothing more.

I am able to verify the functionality, but am at a loss as to what to do in regards to improving the testing of this untestable script (outside of completely rewriting it).

File: scripts/make_2d_plots.py (hunk 1 above)
Comment (ElDeveloper) [inline scripts/make_2d_plots.py]: :scream:

**Comment by ElDeveloper** [comment]
Thanks for the info, it's regrettable to hear this though. Oh well,
let's wait for the tests to pass.

**Comment by gregcaporaso** [comment]
Tested passed before the latest commit, which was just a typo fix.

**Comment by hannahyw** [comment]
Is there any chance this fix changed the Y-axis labels for the scree plot?  When I use make_2d_plots.py with the --scree option passed, I'm getting 0.01 as my highest cumulative fraction of variance, whereas I would expect to get 1 (see below). Not sure if it's an issue with the program, or whether it is truly saying the max cumulative fraction of variance is 1%?  I've tried it on multiple datasets of various quality and am getting the same output. Thanks!

scree_bdiv60up60_unweighted

**Pull request merged**